Sources/Interaction/Widgets/ResliceCursor/example/index.js

```diff
@@ -15,118 +14,98 @@ import vtkRenderWindowInteractor from 'vtk.js/Sources/Rendering/Core/RenderWindo
 // ----------------------------------------------------------------------------
 const container = document.querySelector('body');
 
-// Define Image Data
-
-const imageData = vtkImageData.newInstance();
-const s = 0.1;
-imageData.setSpacing(s, s, s);
-imageData.setExtent(0, 127, 0, 127, 0, 127);
-const dims = [128, 128, 128];
+// Define ResliceCursor
 
-const newArray = new Uint8Array(dims[0] * dims[1] * dims[2]);
+const resliceCursor = vtkResliceCursor.newInstance();
 
-let i = 0;
-for (let z = 0; z < dims[2]; z++) {
-  for (let y = 0; y < dims[1]; y++) {
-    for (let x = 0; x < dims[0]; x++) {
-      newArray[i++] = (256 * (i % (dims[0] * dims[1]))) / (dims[0] * dims[1]);
```

Comment:
This synthetic data could be useful for debug purpose, I would keep it (move into a separate function and do not call it, but keep it)